w3lib/url.py

```diff
@@ -34,7 +34,7 @@ def _quote_byte(error):
 
 _safe_chars = RFC3986_RESERVED + RFC3986_UNRESERVED + EXTRA_SAFE_CHARS + b'%'
 
-def safe_url_string(url, encoding='utf8', path_encoding='utf8'):
+def safe_url_string(url, encoding='utf8', escape_URL=None, path_encoding='utf8'):
```

Comment:
I think ``escape_url`` would be a more conventional name, I think most python APIs don't uppercase it.